cmd/bootstrap/cmd/machine_account.go

```diff
@@ -0,0 +1,177 @@
+
+	"github.com/onflow/flow-go/crypto"
+	model "github.com/onflow/flow-go/model/bootstrap"
+	"github.com/onflow/flow-go/model/flow"
+	ioutils "github.com/onflow/flow-go/utils/io"
+)
+
+var (
+	flagAccessAddress string
+)
+
+// machineAccountCmd represents the `machine-account` command which generates required machine account file
+// for existing and new operators. New operators would have run the `bootstrap keys` cmd get all three keys
+// before running this command.
+var machineAccountCmd = &cobra.Command{
+	Use:   "machine-account",
+	Short: "",
+	Run:   machineAccountRun,
+}
+
+func init() {
+	rootCmd.AddCommand(machineAccountCmd)
+
+	machineAccountCmd.Flags().StringVar(&flagAddress, "address", "", "the node's network address")
+	_ = machineAccountCmd.MarkFlagRequired("address")
+
+	machineAccountCmd.Flags().StringVar(&flagAccessAddress, "access-address", "", "the network address of an access node")
+	_ = machineAccountCmd.MarkFlagRequired("access-address")
+}
+
+// keyCmdRun generate the node staking key, networking key and node information
+func machineAccountRun(_ *cobra.Command, _ []string) {
+
+	// read nodeID written to boostrap dir by `bootstrap key`
+	nodeID, err := readNodeID()
+	if err != nil {
+		log.Fatal().Err(err).Msg("could not read node id")
+	}
+
+	// check if node-machine-account-key.priv.json path exists
+	machineAccountKeyPath := fmt.Sprintf(model.PathNodeMachineAccountPrivateKey, nodeID)
+	keyExists, err := pathExists(filepath.Join(flagOutdir, machineAccountKeyPath))
+	if err != nil {
+		log.Fatal().Err(err).Msg("could not check if node-machine-account-key.priv.json exists")
+	}
+	if !keyExists {
+		log.Fatal().Msg("could not read machine account private key file: run `bootstrap machine-account-key` to create one")
+	}
+
+	// check if node-machine-account-info.priv.json file exists in boostrap dir
+	machineAccountInfoPath := fmt.Sprintf(model.PathNodeMachineAccountInfoPriv, nodeID)
+	infoExists, err := pathExists(filepath.Join(flagOutdir, machineAccountInfoPath))
+	if err != nil {
+		log.Fatal().Err(err).Msg("could not check if node-machine-account-info.priv.json exists")
+	}
+	if infoExists {
+		log.Info().Str("path", machineAccountInfoPath).Msg("node maching account info file already exists")
+		return
+	}
+
+	// read in machine account private key
+	machinePrivKey := readMachineAccountPriv(nodeID)
+	log.Info().Msg("read machine account private key json")
+
+	// create account on access node
+	accountAddress, err := createAccount(machinePrivKey)
+	if err != nil {
+		log.Fatal().Err(err).Msg("could not create account for private key")
+	}
+
+	// create node-machine-account-info.priv.json file
+	machineAccountInfo := assembleNodeMachineAccountInfo(accountAddress.String(), 0, machinePrivKey)
+
+	// write machine account info
+	writeJSON(fmt.Sprintf(model.PathNodeMachineAccountInfoPriv, nodeID), machineAccountInfo)
+}
+
+func createAccount(privateKey crypto.PrivateKey) (sdk.Address, error) {
```

Comment:
⚠️ @danuio  The account is created on Flow Port, not in this CLI. From the [description](https://github.com/dapperlabs/flow-internal/issues/1508#issuecomment-833828753), the machine account is generated in Step 3 in Flow Port, the user copies the address from Flow Port and inputs it to the `machine-account` command (Step 5).

Reply:
ahh yep, I'll remove this